src/rdbms/azext_rdbms/custom.py

```diff
@@ -8,20 +8,75 @@
 from azure.cli.core.commands.client_factory import get_subscription_id
 from msrestazure.tools import resource_id, is_valid_resource_id, parse_resource_id  # pylint: disable=import-error
 from azext_rdbms.mysql.operations.servers_operations import ServersOperations
-from ._client_factory import get_mysql_management_client, get_postgresql_management_client
 
 SKU_TIER_MAP = {'Basic': 'b', 'GeneralPurpose': 'gp', 'MemoryOptimized': 'mo'}
 
 
-# Need to replace source server name with source server id, so customer server restore function
+def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_wait=False,
```

Comment:
Does the method actually use `no_wait`?
Doesn't look like it from the diff.

Reply:
It doesn't.